stixpy/timeseries/quicklook.py

```diff
@@ -342,18 +295,16 @@ def _parse_hdus(cls, hdulist):
             The path to the file you want to parse.
         """
         header = hdulist[0].header
-        control = Table(hdulist[1].data)
-        header["control"] = control
-        data = Table(hdulist[2].data)
-        energies = Table(hdulist[4].data)
+        # control = _hdu_to_qtable(hdulist[1])
```

Comment:
Should this be uncommented?  If not, why is it uncommented in `QLVariance`, below.